packages/type-utils/src/isTypeReadonly.ts

```diff
@@ -1,49 +1,51 @@
-import { ESLintUtils } from '@typescript-eslint/utils';
 import type { JSONSchema4 } from '@typescript-eslint/utils/json-schema';
+
+import { ESLintUtils } from '@typescript-eslint/utils';
 import * as tsutils from 'ts-api-utils';
 import * as ts from 'typescript';
 
-import { getTypeOfPropertyOfType } from './propertyTypes';
 import type { TypeOrValueSpecifier } from './TypeOrValueSpecifier';
+
+import { getTypeOfPropertyOfType } from './propertyTypes';
 import {
   typeMatchesSpecifier,
   typeOrValueSpecifierSchema,
 } from './TypeOrValueSpecifier';
 
 const enum Readonlyness {
-  /** the type cannot be handled by the function */
-  UnknownType = 1,
   /** the type is mutable */
   Mutable = 2,
   /** the type is readonly */
   Readonly = 3,
+  /** the type cannot be handled by the function */
+  UnknownType = 1,
```

Comment:
I **HATE** this.
I'd much rather sort enums by value if they have an explicit value.